packages/trader/src/AppV2/Components/ActiveSymbolsList/active-symbols-list.tsx

```diff
@@ -22,7 +22,8 @@ const ActiveSymbolsList = observer(({ isOpen, setIsOpen }: TActiveSymbolsList) =
     useEffect(() => {
         setSelectedSymbol(symbol);
         setTickData(null);
-    }, [setTickData, symbol]);
+        setDigitStats([]);
```

Comment:
Previously, on symbol change we were not reseting digits stats (they are coming from SmartCharts) ---> while for current spot was shown skeleton-loader, for digit stats - not, they shown old values. Only after we receive new values for current spot, win percentage start updating: first we received '0%' (showing skeleton) and after, when data was recovered, new values.

https://github.com/user-attachments/assets/7cbe77ca-6c80-4aa7-bed8-c310ce22430e

Reply:
I received a request from design team to improve this behavior and show skeleton loader instead of old win percentage values